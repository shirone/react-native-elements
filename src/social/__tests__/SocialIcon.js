import React from 'react';
import {shallow} from 'enzyme';
import toJson from 'enzyme-to-json';
import SocialIcon from '../SocialIcon';

describe('SocialIcon component', () => {
  it('should render without issues', () => {
    const component = shallow(<SocialIcon type="twitter"/>);

    expect(component.length).toBe(1);
    expect(toJson(component)).toMatchSnapshot();
  });
});
