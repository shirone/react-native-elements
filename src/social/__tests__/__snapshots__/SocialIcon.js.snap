// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SocialIcon component should render without issues 1`] = `
<View
  disabled={false}
  onLongPress={[Function]}
  onPress={[Function]}
  style={
    Array [
      Object {
        "shadowColor": "rgba(0,0,0, .4)",
        "shadowOffset": Object {
          "height": 1,
          "width": 1,
        },
        "shadowOpacity": 1,
        "shadowRadius": 1,
      },
      Object {
        "alignItems": "center",
        "borderRadius": 30,
        "flexDirection": "row",
        "justifyContent": "center",
        "margin": 7,
      },
      false,
      Object {
        "height": 52,
        "width": 52,
      },
      false,
      Object {
        "backgroundColor": "#00aced",
      },
      undefined,
      undefined,
    ]
  }
  underlayColor="#00aced"
>
  <View
    style={
      Object {
        "alignItems": "center",
        "flexDirection": "row",
        "justifyContent": "center",
      }
    }
  >
    <Icon
      allowFontScaling={false}
      color="white"
      name="twitter"
      size={24}
      style={
        Array [
          undefined,
        ]
      }
    />
  </View>
</View>
`;
